charts/drax/configuration/grafana/dashboards/5g-cuup-pm-counters.json

```diff
@@ -129,8 +129,8 @@
           "measurement": "uemeasurement.vector",
           "orderByTime": "ASC",
           "policy": "default",
-          "query": "SELECT mean(\"measurement.dlGtpThroughput\") *8 FROM \"uemeasurement.vector\" $timeFilter GROUP BY time(1s) ",
-          "rawQuery": false,
+          "query": "SELECT sum(\"measurement.dlGtpThroughput\") *8 FROM \"uemeasurement.vector\" WHERE $timeFilter GROUP BY time(1s), \"cuupId\"::tag fill(null)",
+          "rawQuery": true,
```

Comment:
I notice we're switching from `mean` to `sum`, and the Panel is called "DL GTP Throughput", but the `alias` mentions "Average throughput". Has the purpose of this panel essentially shifted?

Reply:
I don't know what this `alias` tag is actually used for. Do we need to update something somewhere else to match?